lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -446,61 +448,29 @@ public function render_jwt_auth_integration( $args = array() ) {
 		$this->render_field_description( __( 'This will enable the JWT Auth Users Repository override', 'wp-auth0' ) );
 	}
 
-	// TODO: Deprecate, moved to WP_Auth0_Admin_features
-	public function render_passwordless_enabled( $args = array() ) {
-		$this->render_switch( $args['label_for'], $args['opt_name'] );
-		$this->render_field_description(
-			__( 'Turns on Passwordless login (email or SMS) in the Auth0 form. ', 'wp-auth0' ) .
-			__( 'Passwordless connections are managed in the ', 'wp-auth0' ) .
-			$this->get_dashboard_link( 'connections/passwordless' ) .
-			__( ' and at least one must be active and enabled on this Application for this to work. ', 'wp-auth0' ) .
-			__( 'Username/password login is not enabled when Passwordless is on', 'wp-auth0' )
-		);
+	/**
+	 * @deprecated 3.6.0 - Handled by WP_Auth0_Admin_Features::render_passwordless_enabled()
+	 */
+	public function render_passwordless_enabled() {
+		// phpcs:ignore
+		trigger_error( sprintf( __( 'Method %s is deprecated.', 'wp-auth0' ), __METHOD__ ), E_USER_DEPRECATED );
 	}
 
-	// TODO: Deprecate
+	/**
+	 * @deprecated 3.6.0 - Passwordless method is determined by tenant or Connections setting
```

Comment:
Passwordless method should be first enabled on Connections and then enabled for each Application. No tenants involved AFAIK.

Reply:
> Passwordless method should be first enabled on Connections

Passwordless methods **are** connections:

https://manage.auth0.com/#/connections/passwordless

Still, wording could be improved